### PR TITLE
refresh ed. brand landing page (bug 1103205)

### PR DESCRIPTION
**Pull request by ngokevin**
- app tiles are misaligned, @spasovski is on that
- no background curves yet

screen shot 2015-01-07 at 5 27 31 pm
screen shot 2015-01-07 at 5 27 39 pm

### CODE REVIEW

**Comment by ngokevin** [comment]
@spasovski

File: src/templates/_macros/feed_item.html [hunk 1]
```diff
@@ -39,44 +39,56 @@ <h1 class="name">{{ feed_app.app.name|translate(feed_app) }}</h1>
...
+        {% if loop.index0 < max_apps %}
           {{ app_tile(app, src=src) }}
         {% endif %}
       {% endfor %}
     </ul>
 
     {# Show the 'View all' link if there are 4-5 or 7+ apps in the brand.' #}
-    {% if not landing and ((brand.app_count > 3 and brand.app_count < 6) or
-                            brand.app_count > 6) %}
+    {% if (brand.app_count > 3 and brand.app_count < 6) or brand.app_count > 6 %}
```
Comment (spasovski) [inline src/templates/_macros/feed_item.html]: so if `brand.app_count > 3 and not brand.app_count == 6`

**Comment by spasovski** [comment]
What's up with the missing icons?

**Comment by spasovski** [comment]
We don't have mobile direction on this - and you can leave it to me if you wish but we can't have the header take up this much real-estate on mobile.

**Comment by spasovski** [comment]
Code itself looks good.

r+wc

**Comment by ngokevin** [comment]
icons only show up in a compiled environment. MKT_COMPILE=1 make serve

**Pull request merged**